tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go

```diff
@@ -6,9 +6,11 @@ import (
 	"github.com/rancher/rancher/tests/framework/clients/rancher"
```

Comment:
Apologies for bringing the old stuff, but since this redeclare `name` at [L177](https://github.com/rancher/rancher/blob/2fe09ce5c55613124df6a34033ef8e15ad749393/tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go#L177), the name's gonna be perfectly fine for provisioning tests' [t.Run](https://github.com/rancher/rancher/blob/2fe09ce5c55613124df6a34033ef8e15ad749393/tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go#L181-L183). However since both scale testing and provisioning tests are initialized out of the scope of these for loops at [L165](https://github.com/rancher/rancher/blob/2fe09ce5c55613124df6a34033ef8e15ad749393/tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go#L165), and `scaleName` has never redeclared and combined at [L180](https://github.com/rancher/rancher/blob/2fe09ce5c55613124df6a34033ef8e15ad749393/tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go#L180), the last run of the scale testing's name is gonna be admin user and standard user combined.

Something like this:
`TestRKE1ProvisioningTestSuite/TestProvisioningRKE1ClusterDynamicInput/Admin_User_Node_Provider:_aws_Kubernetes_version:_v1.24.9-rancher1-1_cni:_calico_scalingStandard_User_Node_Provider:_aws_Kubernetes_version:_v1.24.9-rancher1-1_cni:_calico_scaling`

Reply:
I am not quite sure that I understand it sorry Markus. But there's various ways: you can initialize them right before this line `name = tt.name + providerName + " Kubernetes version: " + kubeVersion`. You can update scaleName as `+= `this to `=` for scaleName. There should be many ways to achieve this via a bit experiment. But other point is like I mentioned [above](https://github.com/rancher/rancher/pull/40590#discussion_r1110045017). Since lookarounds aren't supported, it's better not to use the same name with additions but using another subtest name would be sufficient to run single subtest in the table.

So what I mean by that once you run test/name it'll also run test/name_scaling cause it'll match with the name, does that make sense?

Reply:
So Markus what Ugur is saying is that because you are using the `+=` operator without reseting scaleName as it loops the name will get progressively longer. For example 
```
First loop scaleName=“admin client etc.etc_scaling”
Second loop scaleName=“admin client etc.etc_scalingStandardUser client etc.etc._scaling”
```

You see on line177 name it's gets reset. But also because of the regex, since we're just adding a _scaling suffix trying to run just the provisioning will still run scaling, so I think we need rename the scaling something different. I need to give this a little more thought, but I'll get back to you.

Reply:
Yeah, that makes sense to me. I have it changed to Ugur's earlier suggestion of just `=`. I'm going to push a commit soon with all of the changes.

Reply:
We were also talking because of the go's regex capabilities, it would be better to have scaling prefix rather than the suffix. When you run the Jenkins runs can do one where you only run the provisioning and not the scaling?